bin/kano-signal

```diff
@@ -43,6 +46,31 @@ except:
     xml_filename = ""
 
 
+def set_focus(app, verbose=False):
+    '''
+    Sets user input focus to the app name.
+    '''
+    app_titles_dict = {
+        'make-minecraft': 'Make Minecraft',
+        'make-pong': 'Make Pong',
+        'sonicpi': 'Sonic Pi',
+        'make-light': 'Make Light'
+    }
+
+    if app in app_titles_dict:
+        # switch user focus back to the game app.
+        cmd = 'wmctrl -a "{}"'.format(app_titles_dict[app])
+        if verbose: print cmd
+        rc = os.system(cmd)
+        if rc != 0:
+            from kano.logging import logger
+            logger.error("error raising window {}".format(app_titles_dict[app]))
```

Comment:
@tombettany I dont see how this change (which has been merged) is supposed to work. If this were `for app in`, then adding .itervalues() would change `app` to the value instead of the key of the dict. But it's not, it's `if app in`.